common/errors.js

```diff
@@ -6,12 +6,12 @@
     // Factory for each error type
     .factory('ErrorService', ['$log', '$window', '$uibModal', 'UriUtils', 'AlertsService', 'Session', function ErrorService($log, $window, $uibModal, UriUtils, AlertsService, Session) {
 
-        function errorPopup(exception) {
-            $log.info(exception);
+        function errorPopup(message, errorCode, pageName, redirectLink) {
 
```

Comment:
Might be a good idea to have a default `redirectLink`. This would break in other recordedit app, for example, because recordedit doesn't pass in a redirect url when using `errorPopup`.